alohomora/req.py

```diff
@@ -213,11 +216,17 @@ def login_one_factor(self, username, password):
             elif "pass" in name.lower():
                 # Make an educated guess that this is the right field for the password
                 payload[name] = password
+            elif "csrfp_token" in name.lower():
```

Comment:
It needs csrfp_token and AuthState in the header, else it fails with this error

Traceback (most recent call last):
  File "main.py", line 282, in <module>
    Main().main()
  File "main.py", line 196, in main
    (okay, response) = provider.login_two_factor(response, auth_device)
  File "/usr/local/lib/python3.7/site-packages/alohomora/req.py", line 280, in login_two_factor
    sigs = sig_request.split(':')
AttributeError: 'NoneType' object has no attribute 'split'